webapp/src/ts/services/contact-view-model-generator.service.ts

```diff
@@ -309,7 +311,7 @@ export class ContactViewModelGeneratorService {
     });
     const filter = { subjectIds: _flattenDeep(subjectIds) };
     return this.searchService
-      .search('reports', filter, { include_docs: true })
+      .search('reports', filter, { include_docs: true, limit: this.LIMIT_SELECT_ALL_REPORTS })
```

Comment:
I think 500 is fine, we already use the limit of 500 in the selectall reorts (https://github.com/medic/cht-core/blob/c4c3114ed8e17f49a78e1f803a643792b71f2c8e/webapp/src/ts/modules/reports/reports.component.ts#L67).